app/assets/javascripts/account/AccountController.js.coffee

```diff
@@ -37,14 +37,14 @@ AccountController = (
   $scope.passwordRegex = /^(?=.*[0-9])(?=.*[a-zA-Z])(.+){8,}$/
   $scope.emailRegex = SharedService.emailRegex
   $scope.showSaleListings = $window.env.showSaleListings == 'true'
-  $scope.passwordConfirmation
 
   $scope.accountForm = ->
     # pick up which ever one is defined (the other will be undefined)
     $scope.form.signIn ||
     $scope.form.createAccount ||
     $scope.form.updatePassword ||
     $scope.form.passwordReset ||
+    $scope.form.accountPassword ||
```

Comment:
@software-project this then messes with updating values in other forms on the account settings page. I tried it yesterday evening

Reply:
@akegan I tested all other forms on account settings and they are working as expected. Reset password as well. It didnt work for me when I put that line as the last element. Maybe that was the case?

Reply:
Looks like just need to remove this line and otherwise looks good!
```suggestion
```